enterprise-metrics/main.libsonnet

```diff
@@ -538,9 +549,13 @@ local removeNamespaceReferences(args) = std.map(function(arg) std.strReplace(arg
     createSecretContainer::
       container.new('create-secret', this._images.kubectl)
       + container.withCommand([
-        '/bin/bash',
-        '-euc',
-        'kubectl create secret generic gem-admin-token --from-file=token=/shared/admin-token --from-literal=grafana-token="$(base64 <(echo :$(cat /shared/admin-token)))"',
+        'kubectl',
```

Comment:
This is much easier to read, would you be able to double check this still works as expected, perhaps using the test tooling in #598?

Reply:
Sure, once you merged I can rebase :)